scripts/pip-post-compile.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+set -e
+
+bash scripts/_vendor/pip-post-compile.sh "$@"
```

Comment:
Somewhat unrelated to this PR, but it looks like this file is generated by curling github and then saving it outside of our git repo. Is there a reason why we aren't including that file in our repository?

Reply:
Yeah good question. You can follow my reasoning in this PR description: https://github.com/dimagi/commcare-hq/pull/21490#issue-207163955.

Reply:
Your reasoning makes sense. I do have an aversion to `curl | bash` but this is about the same as installing packages from specific commits/versions

Reply:
Yeah, I know what you mean. The nice things I considered are that it's over https, so it's not going to be man-in-the-middled, and the url contains the commit hash, so it's not going to change out from under us. The only way (that my little brain can think of) for us to get and execute the wrong thing is for github's infrastructure to get compromised, which is the same failure mode as checking out our own code and running it in production.